cronjob/templates/cronjob.yaml

```diff
@@ -35,10 +37,10 @@ spec:
                 - name: {{ $key }}
                   value: {{ $value | quote }}
                 {{- end }}
-                {{- range $key, $name := .Values.envSecrets }}
+                {{- range $key, $ref := .Values.envSecrets }}
                 - name: {{ $key }}
                   valueFrom:
                     secretKeyRef:
-                      name: {{ $name }}
-                      key: {{ $key | quote }}
+                      name: {{ $ref.name }}
+                      key: {{ $ref.key | quote }}
```

Comment:
i was intended to make the key same as the key in key ref as it's basically always the same